app/Http/Controllers/LoginController.php

```diff
@@ -12,11 +12,10 @@ public function __construct(private LoginRepository $loginRepository)
 
     public function login(LoginRequest $request)
     {
+        $errors = ["Login Credentials are invalid"];
+
         if(! $this->loginRepository->login($request->validated()) ) {
             return redirect()->route('login')
-                ->with("message", "Login failed");
-        }
-
-        return redirect()->route('dashboard.index');
```

Comment:
why are you removing this redirect to dashboard?

Reply:
I am not removing it, it's still in the code